meta/template/jit.lua

```diff
@@ -26,7 +26,7 @@ function jit.off(func, recursive) end
 function jit.flush(func, recursive) end
 
 ---@return boolean status
----@return ...
+---@return any ...
```

Comment:
Minor correction from the [luajit docs](https://luajit.org/ext_jit.html):

> Returns the current status of the JIT compiler. The first result is either true or false if the JIT compiler is turned on or off. The remaining results are strings for CPU-specific features and enabled optimizations.

```suggestion
---@return string ...
```